client/components/jetpack-header/docs/example.js

```diff
@@ -21,6 +21,7 @@ export default class JetpackHeaderExample extends PureComponent {
 				<JetpackHeader partnerSlug="bluehost" />
 				<JetpackHeader partnerSlug="inmotion" />
 				<JetpackHeader partnerSlug="liquidweb" />
+				<JetpackHeader partnerSlug="eurodns" />
```

Comment:
Quick bit of feedback, if we want to cobrand, then we'll also need to get their logo and create a component within `JetpackHeader`. Look at the `JetpackHeader` component for some examples.

Reply:
Added logo for cobranding